src/main/webapp/secured/users.xhtml

```diff
@@ -75,51 +75,7 @@
                     </div>
                     <div class="ui-g-2">
                     </div>
-                </div>
-                <h3>Create a new user</h3>
-                <div class="ui-g">
-                    <div class="ui-g-2">
-                    </div>
-                    <div class="ui-g-8">
-                        <h:form id="add">
-                            <h:panelGrid columns="3" cellpadding="5" style="margin: 0 auto;">
```

Comment:
add  <p:growl id="growl" sticky="true"/> here so you are sending "add succesfull" message check how we did for invoice and customer view